rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -221,8 +256,8 @@ def get_balances(
                     if exist_result is None or bool(exist_result[0]) is False:
                         cursor.execute(
                             'INSERT OR IGNORE INTO nfts('
-                            'identifier, name, last_price, last_price_asset, manual_price, owner_address'  # noqa: E501
```

Comment:
Here you are doing an `INSERT_OR_IGNORE` but if I understood correctly if we query opensea we might want to update the prices and image_url at least. Shouldn't it be a `INSERT OR REPLACE` in that case?

Reply:
Yeah, makes sense :+1: